### PR TITLE
376 color update to the new brand pallette

### LINKED ISSUE #376: Color: update colors to new brand colors
We need to update the colors to new hex values based on new brand colors which were just released.

The colors are as follows:

**Links**
Link: #3665F3 (B4)
Link hover: #121258 (B7)

**Notice**
Information: #3665F3 (B4)
Confirmation: #3AC952 (G4)
Exclamation: #E62048 (R4)

Information text: #3665F3 (B4)
Confirmation text: #3AC952 (G6)
Exclamation text: #E62048 (R4)

new brand colors

### PR DESCRIPTION
**Pull request by seangates**
## Description
- update color variables
- new SVG for notice icons based on new colors

## Context
Brand colors were updated, causing Skin to need to update.

## References
Resolves #376 

## Screenshots
localhost_3000_ds6_ 1

image

image

### CODE REVIEW

File: dist/less/ds6/variables.less [hunk 1]
```diff
@@ -3,43 +3,98 @@
 // Colours
 //----------------------
 
-@ds6-color-g201-gray: #fff;
```
Comment (ianmcburnie) [inline dist/less/ds6/variables.less]: We should leave these variables in place for backwards compatibility.

File: src/less/button/ds6/button-cta.less [hunk 2]
```diff
@@ -16,61 +16,61 @@ a.cta-btn {
     }
 }
 
-a.cta-btn--p002-blue {
```
Comment (ianmcburnie) [inline src/less/button/ds6/button-cta.less]: This is a breaking change. We need to leave these old names in place for a minor version. We can remove them in 7.0.0.

File: dist/less/ds6/variables.less (hunk 1 above)
Comment (ianmcburnie) [reply dist/less/ds6/variables.less]: When adding them back in, please also add a comment that they are deprecated.

File: src/less/button/ds6/button-cta.less (hunk 2 above)
Comment (seangates) [reply src/less/button/ds6/button-cta.less]: I actually thought of that _just_ as I was submitting the PR. Already in the works.

File: dist/less/ds6/variables.less (hunk 1 above)
Comment (seangates) [reply dist/less/ds6/variables.less]: I actually thought of that just as I was submitting the PR. Already in the works.

**Comment by ianmcburnie** [comment]
I find it a little odd that we got rid of `#767676`, which is the lightest we can go on a white background for a11y colour contrast.

**Comment by seangates** [comment]
@ianmcburnie Updated per PR feedback and today's discussions. Please check again.

**Comment by seangates** [comment]
@ianmcburnie Rebased with the combobox/listbox updates;

**Pull request merged**